KarhooUISDK/API/KarhooComponents.swift

```diff
@@ -27,7 +27,7 @@ public class KarhooComponents: BookingScreenComponents {
         presenter.load(view: addressBarView)
 
         if let journey = journeyInfo {
-            KarhooBookingStatus.shared.setJourneyInfo(journeyInfo: journey)
+            KarhooJourneyDetailsController.shared.setJourneyInfo(journeyInfo: journey)
```

Comment:
As you imagine, the `Controller` name is a bit risky in our iOS environment for obvious reason. I am ok with that name, but still think anything (handler, manager) would allow us to avoid any potential misunderstanding on DP side.

Reply:
I think it could be misleading even for us.

Reply:
Changed it to `Manager`